package-lock.json

```diff
@@ -1,8 +1,3231 @@
 {
   "name": "alpha-site",
   "version": "0.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
```

Comment:
Not sure why this keeps happening to your PRs, @will0684. The actual PR template looks fine, but we definitely don't want to include 3000+ lines in the package-lock file as we didn't add any new packages here.

Reply:
I agree. I'll try to figure out what's going on here.

Reply:
could you be using node 14 as opposed to 12 ?

Reply:
Yes I am using 15 it seems, I'll switch down to 12 for the time being.